packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-run-sample/main/java/testapp.md

```diff
@@ -4,11 +4,12 @@
 
 1. Ensure that you've [set up your Okta org for one password factor only use cases](/docs/guides/oie-embedded-common-org-setup/java/main/#set-up-your-okta-org-for-password-factor-only-use-cases).
 
-1. Obtain app-specific configuration values (such as `{clientId}`, `{clientSecret}`, `{yourOktaDomain}`) from the app integration you've created in [Create new application](/docs/guides/oie-embedded-common-org-setup/java/main/#create-a-new-application).
+1. Obtain app-specific configuration values (such as `{clientId}`, `{clientSecret}`, `{yourOktaDomain}`) from the [app integration](/docs/guides/oie-embedded-common-org-setup/java/main/#create-a-new-application) you've created in Okta.
```

Comment:
```suggestion
1. Obtain app-specific configuration values (such as `${clientId}`, `${clientSecret}`, `${yourOktaDomain}`) from the [app integration](/docs/guides/oie-embedded-common-org-setup/java/main/#create-a-new-application) you've created in Okta.
```

Reply:
Please refer to URL style: https://github.com/okta/okta-developer-docs/wiki/Style-Guide